test

```diff
@@ -0,0 +1 @@
+test
```

Comment:
what is this file for?

Reply:
I have no idea how I added this file. I'll remove it, thank you Marco for noticing it.